datadog/data_source_datadog_metric_tags.go

```diff
@@ -0,0 +1,59 @@
+package datadog
+
+import (
+	"context"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
```

Comment:
We don't allow new datasources (or resources) using the sdk. Every new one must be implemented using the provider framework. You can find examples under the `/datadog/fwprovider` directory.

Reply:
@fpighi Thanks; apologies for overlooking that. I've pushed a new commit doing so.